src/main/java/com/geekbeast/rhizome/registries/ObjectMapperRegistry.java

```diff
@@ -46,7 +46,7 @@ protected static ObjectMapper createSmileMapper() {
     }
 
     protected static ObjectMapper createPlainMapper() {
-        ObjectMapper mapper = new ObjectMapper( new SmileFactory() );
+        ObjectMapper mapper = new ObjectMapper();
         mapper.registerModule( new GuavaModule() );
```

Comment:
+1 on the JodaModule. Just to clarify, we don't want the SmileFactory used here because we have a separate SmileMapper right?